fe-nc-news/src/AddComment.jsx

```diff
@@ -0,0 +1,62 @@
+import {React, useState, useEffect} from 'react'
+import { postComment } from "./utils/ApiFunctions";
+import "./styles/articles.css"
+
+const AddComment = ({article_id}) => {
```

Comment:
We might want to pass a state that allows us to update all the comments, at the moment when we write two new comments, the first one we wrote will not be rendered 🙂